### PR TITLE
RAIL-3767 FilterBar and dashboard content style fixes

### PR DESCRIPTION
**Pull request by plavemar**
- FilterBar and dashboard content style fixes

JIRA: RAIL-3767

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/DefaultFilterBarContainer.tsx [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+    selectFilterBarHeight,
+    selectLocale,
+    uiActions,
+    useDashboardDispatch,
+    useDashboardSelector,
+} from "../../../model";
 import { ShowAllFiltersButton } from "./ShowAllFiltersButton";
 
 // TODO: this will probably need to be customizable so that custom filter components work
 const DEFAULT_FILTER_BAR_HEIGHT = 58;
```
Comment (no23reason) [inline libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/DefaultFilterBarContainer.tsx:17]: Please extract this constant to some place in presentation/constants

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5369/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7087/ : SUCCESS in 16m 09s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1527/ : SUCCESS in 1m 09s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1807/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/597/ : SUCCESS in 18m 20s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7089/ : FAILURE in 7m 13s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Pull request merged**